### PR TITLE
"pages needing transcription" button now fits the kind of project it is.

### PR DESCRIPTION
**Pull request by JosieBrum**
All transcription projects have "pages needing transcription". All correction projects have "pages needing correction". Mixed projects have "pages needing correction or transcription".

### CODE REVIEW

File: spec/features/collection_spec.rb [hunk 1]
```diff
@@ -240,7 +240,7 @@
     visit collection_path(@collection.owner, @collection)
     expect(page).to have_content("About")
     expect(page).to have_content("Works")
-    page.click_link("Pages That Need Transcription")
+    page.click_link("Pages That Need Correction or Transcription")
```
Comment (saracarl) [inline spec/features/collection_spec.rb:243]: Can we externalize these messages?  We'll need a way to mark them in the dictionary file as "needing translation" -- maybe with a comment that the translator can remove once it's translated?

File: app/views/collection/show.html.slim [hunk 2]
```diff
@@ -80,8 +80,12 @@
...
         
     -if @collection.active?
-      =link_to "Pages That Need Transcription", collection_needs_transcription_path(@collection.owner, @collection), class: 'button review-button'
-
+      -if @collection.works.where(:ocr_correction => true).present? && @collection.works.where(:ocr_correction => false).present?
+        =link_to "Pages That Need Correction or Transcription", collection_needs_transcription_path(@collection.owner, @collection), class: 'button review-button'
+      -elsif @collection.works.where(:ocr_correction => true).present?
+        =link_to "Pages That Need Correction", collection_needs_transcription_path(@collection.owner, @collection), class: 'button review-button'
+      -else
+        =link_to "Pages That Need Transcription", collection_needs_transcription_path(@collection.owner, @collection), class: 'button review-button'
```
Comment (saracarl) [inline app/views/collection/show.html.slim:88]: Can we externalize these messages?  We'll need a way to mark them in the dictionary file as "needing translation" -- maybe with a comment that the translator can remove once it's translated?

File: app/views/collection/show.html.slim [hunk 3]
```diff
@@ -80,8 +80,12 @@
       =label_tag 'search', "Search for works", class: 'hidden'
         
     -if @collection.active?
-      =link_to "Pages That Need Transcription", collection_needs_transcription_path(@collection.owner, @collection), class: 'button review-button'
-
+      -if @collection.works.where(:ocr_correction => true).present? && @collection.works.where(:ocr_correction => false).present?
+        =link_to "Pages That Need Correction or Transcription", collection_needs_transcription_path(@collection.owner, @collection), class: 'button review-button'
```
Comment (saracarl) [inline app/views/collection/show.html.slim:84]: Can we externalize these messages?  We'll need a way to mark them in the dictionary file as "needing translation" -- maybe with a comment that the translator can remove once it's translated?

File: app/views/collection/show.html.slim [hunk 4]
```diff
@@ -80,8 +80,12 @@
...
+      -elsif @collection.works.where(:ocr_correction => true).present?
+        =link_to "Pages That Need Correction", collection_needs_transcription_path(@collection.owner, @collection), class: 'button review-button'
```
Comment (saracarl) [inline app/views/collection/show.html.slim:86]: Can we externalize these messages?  We'll need a way to mark them in the dictionary file as "needing translation" -- maybe with a comment that the translator can remove once it's translated?

File: app/views/collection/show.html.slim (hunk 2 above)
Comment (saracarl) [reply app/views/collection/show.html.slim:88]: Actually, I think we have an existing message for this one, and maybe for the correction one.

**Review by saracarl — CHANGES_REQUESTED** [review]
Needs I18N

**Comment by benwbrum** [comment]
Code migrated to PR #3099.  Closing PR.

**Pull request closed without merging**